sources/Application/Views/InstrumentView.cpp

```diff
@@ -84,125 +84,110 @@ void InstrumentView::fillSampleParameters() {
   position._y -= 1;
   Variable *v = instrument->FindVariable(SIP_SAMPLE);
   SamplePool *sp = SamplePool::GetInstance();
-  intVarField_.emplace_back(UIIntVarField(position, *v, "sample: %.19s", 0,
-                                          sp->GetNameListSize() - 1, 1, 0x10));
+  intVarField_.emplace_back(position, *v, "sample: %.19s", 0,
```

Comment:
How does this work? as according to the docs: https://www.etlcpp.com/vector.html `emplace()` only supports up to 4 constructor params?

Reply:
@maks most likely typo, check for example https://www.etlcpp.com/list.html, it clarifies 4 arguments for c++03 but no limit for c++11, and same for other containers.